src/App.tsx

```diff
@@ -1,24 +1,287 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import { Todo } from './types/Todo';
+import {
+  addTodos,
+  deleteTodos,
+  getTodos,
+  updateTodos,
+} from './api/todos';
+import { Header } from './Components/Header';
+import { TodoList } from './Components/Todolist';
+import { Footer } from './Components/Footer';
+import { Notification } from './Components/Notification';
+import { Filters } from './types/enums';
+import { AppContext } from './AppContext';
+import { LoginForm } from './Components/LoginForm';
 
-const USER_ID = 0;
+const temp = {
+  id: 0,
+  userId: 0,
+  title: '',
+  completed: false,
+};
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [selectedStatus, setSelectedStatus] = useState(Filters.all);
+  const [errorMessage, setErrorMessage] = useState('');
+  const hasErrorFromServer = !!errorMessage;
+  const [deletedId, setDeletedId] = useState(0);
+  const [tempTodo, setTempTodo] = useState<Todo>(temp);
+  const [added, setAdded] = useState(false);
+  const [updatedId, setUpdatedId] = useState<number | string>(0);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
It's better to set the initial value for isLoading to false and change it to true when you actually start loading
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```